compiler+runtime/src/cpp/jank/runtime/obj/detail/iterator_sequence.cpp

```diff
@@ -39,7 +39,7 @@ namespace jank::runtime::obj::detail
             return false;
           }
         }
-        return true;
+        return seq == nullptr;
```

Comment:
Very odd that you made the opposite change here that you did in other places. I think all of them should be `return seq == nullptr;`